app/code/Magento/QuoteGraphQl/Model/Cart/AddSimpleProductToCart.php

```diff
@@ -140,21 +150,45 @@ private function extractCustomizableOptions(array $cartItemData): array
         return $customizableOptionsData;
     }
 
+    /**
+     * @param $product
+     * @param array $cartItemData
+     * @return array
+     */
+    private function extractDownloadableLinks($product, array $cartItemData): array
```

Comment:
All related to `Downloadable product` logic should be moved to `DownloadableGraphQl` module

Reply:
**done**